src/utils/client-url.ts

```diff
@@ -0,0 +1,34 @@
+import ClientRepresentation from "@keycloak/keycloak-admin-client/lib/defs/clientRepresentation";
+import { joinPath } from "./joinPath";
+
+export const convertClientToUrl = (
+  client: ClientRepresentation,
+  adminClientBaseUrl: string
+) => {
+  const rootUrl = client.rootUrl;
+  const baseUrl = client.baseUrl;
```

Comment:
Perhaps we can clean this up a bit with some destructuring?
```suggestion
export const convertClientToUrl = (
  { rootUrl, baseUrl }: ClientRepresentation,
  adminClientBaseUrl: string
) => {
```